Android.mk

```diff
@@ -120,8 +120,12 @@ LOCAL_JAVA_LANGUAGE_VERSION := 1.7
 include $(BUILD_STATIC_JAVA_LIBRARY)
 
 bundled_test_java_files := $(filter-out \
+  %/org/conscrypt/AbstractSessionContextTest.java \
+  %/org/conscrypt/ClientSessionContextTest.java \
```

Comment:
nit: not sure if github is displaying this wrong or if there is missing whitespace here.